index.js

```diff
@@ -1,41 +1,46 @@
-
-var props = [
+var defaultProps = [
   'name',
   'message',
   'stack',
-  'type',
-]
+  'type'
+];
+
+module.exports = function (propertiesToShow) {
+
+  var props = propertiesToShow || defaultProps;
 
-module.exports = function () {
   return function* jsonErrorHandler(next) {
-    var status
+    var status;
```

Comment:
Probably shouldn't add a semi-colon unless you are going to add them everywhere; however, I think that is counter to this particular project's style.